MixpanelDemo/MixpanelDemo.xcodeproj/xcshareddata/xcschemes/MixpanelDemoWatch (Notification).xcscheme

```diff
@@ -56,16 +56,18 @@
       allowLocationSimulation = "YES"
       launchAutomaticallySubstyle = "8"
       notificationPayloadFile = "MixpanelDemoWatch Extension/PushNotificationPayload.apns">
-      <BuildableProductRunnable
-         runnableDebuggingMode = "0">
+      <RemoteRunnable
```

Comment:
you probably didn't want these. i don't know why Xcode always wants to change this particular scheme.

Reply:
yeah, the new Xcode... it's annoying. We should probably add gitinore for the scheme files.